tests/unit/date/test_datetime_to_ms_roundtrip.py

```diff
@@ -45,6 +46,8 @@ def test_pytz_London():
     assert_roundtrip(tz)
 
 
+@pytest.mark.xfail(sys.version_info >= (3,),
+                   reason="known issue with dateutil and python3")
```

Comment:
FYI, I tracked this down and it is an issue with Python that should be OK for Python 3.4.4+.